src/Bookings.js

```diff
@@ -1,18 +1,61 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import Search from "./Search.js";
-// import SearchResults from "./SearchResults.js";
-// import FakeBookings from "./data/fakeBookings.json";
+import SearchResults from "./SearchResults.js";
+//import FakeBookings from "./data/fakeBookings.json";
 
 const Bookings = () => {
+  const [bookings, setBookings] = useState([]);
+  const [filteredBookings, setFilteredBookings] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [error, setError] = useState(false);
+
+  useEffect(() => {
+    setIsLoading(true);
+    fetch("https://cyf-react.glitch.me")
+      .then(res => {
+      if(res.status >= 400) {
+        throw new Error("Server responds with error!");
+      } else {
+        return res.json()
+      }})
+      .then(data => {
+        setBookings(data);
+        setIsLoading(false);
+    },
+    err => {
+      setError(err)
+      setIsLoading(false);
+    });
+  }, []);
+
+  useEffect(()=> {
+    setFilteredBookings(bookings)
+  }, [bookings])
+
+
   const search = searchVal => {
-    console.info("TO DO!", searchVal);
+   if(searchVal) {
+     const filteredBookings = bookings.filter(booking => {
+       const {firstName, surname} = booking;
+       const preparedSearchVal = searchVal.toLowerCase();
+       const preparedFirstName = firstName.toLowerCase();
+       const preparedSurname = surname.toLowerCase();
+       return preparedFirstName.includes(preparedSearchVal) || preparedSurname.includes(preparedSearchVal)
+     });
+     setFilteredBookings(filteredBookings);
+   } else {
+     setFilteredBookings(bookings);
+   }
   };
 
+
   return (
     <div className="App-content">
       <div className="container">
         <Search search={search} />
```

Comment:
Can you think how you'd modify this code so that the search function lives in the search component?

Some questions that might help you think about a way you could are:
1. Is the bookings state needed anywhere else other than the search?
2. Which setState functions would you need to pass down into the search component